### DISCUSSION TITLE
Issue with resources

### DISCUSSION THREAD

**Discussion by Elip100** [geode-sdk/sdk#1002] [Help]
I am new to Geode and C++ and can't figure out why this mod doesn't work. I followed the documentation. It only works when using custom textures on the button.
[mod.zip](https://github.com/user-attachments/files/16239560/mod.zip)
(I deleted the build folder to stay under github file size limit)

**Comment by Fleyzix76** [comment]
I have almost the same problem, but only in .github/workflows/multi-platform.yml I can not make my mod work on android64.32, windows, macOS and ios.

**Comment by HenrySck075** [comment]
using discussions for things like this is insane (this is old ik)
